monitoring/tls/grafana-datasource-prom-https.yaml

```diff
@@ -8,3 +8,9 @@
         tlsSkipVerify: true
       editable: true
       url: https://v4m-prometheus:9090
+    - name: Tempo
+      type: tempo
+      url: http://v4m-tempo:3100
+      isDefault: false
+      editable: true
+      apiVersion: 1
```

Comment:
See previous notes questioning whether we need/want to put Tempo datasource definition yaml inside the Prometheus datasource definition.  Are all 3 Tempo datasource definitions identical?